lib/local-links-manager/import/enabled_service_checker.rb

```diff
@@ -25,14 +33,41 @@ def enable_services
 
     private
 
-      def set_enabled_service(row)
-        service = Service.find_by(lgsl_code: row["LGSL"])
-        if service.nil?
-          Rails.logger.warn("'#{row['LGSL']}' is not an imported Service")
-        else
-          service.enabled = true
-          Rails.logger.info("'#{row['LGSL']}' enabled")
-          service.save!
+      def supported_lgsl_codes
```

Comment:
I was trying to tidy up `enable_services` a bit, that's all.  No real preference either way.